operations/kubernetes/Determine_if_Pods_are_Hitting_Resource_Limits.md

```diff
@@ -1,134 +1,106 @@
 # Determine if Pods are Hitting Resource Limits
 
-Determine if a pod is being CPU throttled or hitting its memory limits \(OOMKilled\). Use the detect\_cpu\_throttling.sh script to determine if any pods are being CPU throttled, and check the Kubernetes events to see if any pods are hitting a memory limit.
+Determine if a pod is being CPU throttled or hitting its memory limits (`OOMKilled`).
+Use the `/opt/cray/platform-utils/detect_cpu_throttling.sh` script to determine if any pods are being CPU throttled, and check the Kubernetes events to see if any pods are hitting a memory limit.
 
-**IMPORTANT:** The presence of CPU throttling does not always indicate a problem, but if a service is being slow or experiencing latency issues, this procedure can be used to evaluate if it is not performing well as a result of CPU throttling.
+**IMPORTANT:** The presence of CPU throttling does not always indicate a problem, but if a service is being slow or experiencing latency issues,
+this procedure can be used to evaluate if it is not performing well as a result of CPU throttling.
 
 Identify pods that are hitting resource limits in order to increase the resource limits for those pods.
 
-### Prerequisites
+## Prerequisites
 
 `kubectl` is installed.
 
+## Procedure
 
-### Procedure
+1. Use the `/opt/cray/platform-utils/detect_cpu_throttling.sh` script to determine if any pods are being CPU throttled _(this script is installed on master and worker NCNs)_.  The script can be used in two different ways:
 
-1.  Use the detect\_cpu\_throttling.sh script to determine if any pods are being CP throttled.
+    1. (`ncn-mw#`) Pass in a substring of the desired pod names.
```

Comment:
Need to switch this over to the 1.2 command prompt conventions.

Reply:
fixed!